xbmc/interfaces/AnnouncementManager.h

```diff
@@ -90,6 +91,6 @@ namespace ANNOUNCEMENT
 
     CCriticalSection m_announcersCritSection;
     CCriticalSection m_queueCritSection;
-    std::vector<IAnnouncer *> m_announcers;
+    std::unordered_map<IAnnouncer*, int> m_announcers;
```

Comment:
unordered? So far by using a vector order of time of registration was kept. Not that this change causes hard to catch side effects?!

Reply:
I don't think so, there is no order/synchronism in this interface